Buildings/Resources/Scripts/Dymola/ThermalZones/EnergyPlus/Validation/SurfaceComparison/ZoneSurfaces.mos

```diff
@@ -0,0 +1,13 @@
+﻿simulateModel("Buildings.ThermalZones.EnergyPlus.Validation.SurfaceComparison.ZoneSurfaces", stopTime=2592000, method="Cvode", tolerance=1e-06, resultFile="ZoneSurfaces");
+simulateModel("Buildings.ThermalZones.EnergyPlus.Validation.SurfaceComparison.BaseClasses.ReferenceSurfaces", stopTime=2592000, method="Cvode", tolerance=1e-06, resultFile="ReferenceSurfaces");
```

Comment:
This won't work in our CI testing, as we can only have one `simulateModel` command. You can work around this by making a top-level model that instantiates the two models that you want to compare, as in `Buildings.ThermalZones.EnergyPlus.Validation.MultipleBuildings.ThreeZonesTwoBuildings`

Reply:
Done. Combined both building under `Buildings.ThermalZones.EnergyPlus.Validation.SurfaceComparison.SurfaceComparison`.